core/store/src/config.rs

```diff
@@ -59,6 +59,9 @@ pub struct StoreConfig {
     /// This config option is temporary and will be removed once flat storage is implemented.
     pub sweat_prefetch_senders: Vec<String>,
 
+    pub claim_sweat_prefetch_config: Vec<PrefetchConfig>,
+    pub kaiching_prefetch_config: Vec<PrefetchConfig>,
+
```

Comment:
nit: It would be nice to put all of the prefetch configs into a dedicated struct e.g. `PrefetchConfig`. The only concern is that if any nodes have this config overrided they may not notice and lose their configuration. Do you know if that is the case? Would it be sufficient to announce this change in the CHANGELOG? 

It would make the code nicer but is a bit annoying so up to you if you want to do it.